.github/workflows/update-gas-info.yaml

```diff
@@ -0,0 +1,53 @@
+name: 'Update Gas Info on Testnet'
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: Environment to update gas info on
+        required: true
+        default: testnet
+        type: choice
+        options:
+          - testnet
+          - mainnet
+          - stagenet
+
+  schedule:
+    - cron: '0 */2 * * *'
+
```

Comment:
```suggestion
```
thinking of removing the cron for now until we add some optimizations and make it a bit more reliable. we can update manually until then if needed